lib/bolt/cli.rb

```diff
@@ -261,11 +261,18 @@ def execute(options)
         screen += '_object'
       end
 
-      @analytics.screen_view(screen,
-                             output_format: config.format,
-                             target_nodes: options.fetch(:targets, []).count,
-                             inventory_nodes: inventory.node_names.count,
-                             inventory_groups: inventory.group_names.count)
+      screen_view_fields = {
+        output_format: config.format,
+      }
+
```

Comment:
The unrelated failure in travis has been resolved (see https://github.com/puppetlabs/bolt/pull/985 conversation for details). There are a couple rubocop violations. I dont think we would likely regress on this but perhaps a test would make sense?

Reply:
Hmm I can look into how to test this.